tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -274,11 +277,13 @@ take a look over the official
   So when using e.g. ``CallDeferred("AddChild")``, ``AddChild`` will not work because
   the API is expecting the original ``snake_case`` version ``add_child``. However, you
   can use any custom properties or methods without this limitation.
+  Prefer using the exposed StringName in the ``PropertyName``, ``MethodName`` and
+  ``SignalName`` to avoid extra StringName allocations and worrying about snake_case naming.
```

Comment:
```suggestion
  Prefer using the exposed ``StringName`` in the ``PropertyName``, ``MethodName`` and
  ``SignalName`` to avoid extra ``StringName`` allocations and worrying about snake_case naming.
```

Or is there a reason to avoid formatting it here?

Reply:
Not really, I probably thought it looked too crowded.